microservices/front-end/src/components/auth/AuthRegister.tsx

```diff
@@ -115,8 +131,31 @@ const AuthRegister: FC<AuthRegisterProps> = ({
           variant="outlined"
           fullWidth
           value={confirmPassword}
-          onChange={(e) => onConfirmPasswordChange(e.target.value)}
+          onChange={(e) => {
+            onConfirmPasswordChange(e.target.value);
+            onErrorChange(null);
+          }}
         />
+
+        {/* Handle error situation */}
+        {error && (
+          <Box mt={2}>
+            <Alert severity="error" onClose={() => onErrorChange(null)}>
+              <AlertTitle>Registeration Error</AlertTitle>
+              {error}
+            </Alert>
+          </Box>
+        )}
```

Comment:
When the error alert shows (e.g., the user tries to sign up with a duplicated username), the page overflows vertically in an unintended way. One solution is to use CSS `display: flex` on the parent element and CSS `margin: auto` on the child element

(If you don't know what I mean, I can commit the `display: flex` and `margin: auto` changes for you)

Reply:
To be honest I am a bit stuck at this part. I tried to add display:flex in Registers Card, and margin: auto at Register inner-layer Card; and add display: flex at the wrapper of AuthRegister, and margin:auto at AuthRegister's box (which messed up the text field so I delete). However, the overflow persists. In the end, I adjusted the margin between text boxes 5px down each, to superficially resolved this issue. May I know how you would resolve it?